xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESink.cpp

```diff
@@ -540,7 +540,7 @@ void CActiveAESink::StateMachine(int signal, Protocol *port, Message *msg)
           delete m_sink;
           m_sink = nullptr;
           m_state = S_TOP_CONFIGURED_SUSPEND;
-          m_extTimeout = 10000;
+          m_extTimeout = 10000ms;
```

Comment:
Does C++17 have a separator that lets you do something like 10_000ms?

Reply:
otherwise, representing as 10s might be more clear

Reply:
Thanks! I've updated all 5 digit values to use the next size unit.